### DISCUSSION TITLE
Troubles with preprocessing of proof

### DISCUSSION THREAD

**Discussion by abdulrahman-hegazy** [community/community#30184] [GitHub Education]
I tried many times to send proof of my studing but i always get error while preprocessing (Your image should include the name of your school). But there is a name of my school in doc, but I think trouble causes that it's written on arabic. What can I do?

The actual message

```
The image you selected does not appear to contain your school name. You may include multiple documents in your image, so long as they are legible.
If you meant to apply as a student, please reapply here.
Your image should include the name of your school

**Comment by NaiaraSandi1995** [comment]
Eu não sei.
